includes/orders/functions/refunds.php

```diff
@@ -373,9 +373,13 @@ function edd_refund_order( $order_id = 0, $order_items = array() ) {
 		? 'refunded'
 		: 'partially_refunded';
 
-	edd_update_order( $order_id, array(
-		'status' => $order_status,
-	) );
+	edd_update_order(
+		$order_id,
+		array(
+			'status'      => $order_status,
+			'customer_id' => $order->customer_id,
```

Comment:
Why is this here now?

Reply:
If the `customer_id` is not passed through to the `edd_update_order` function, the customer's stats are not updated (commit [cc4f12c](https://github.com/easydigitaldownloads/easy-digital-downloads/commit/cc4f12ce372fc4127599ae228e41cc3218665e49))

Reply:
Ah I see. Makes sense. Thanks!